### PR TITLE
📊 nasa: ozone layer

### PR DESCRIPTION
**Pull request by lucasrodes**
[Tracking issue](https://github.com/owid/owid-issues/issues/1411)
Adds a new datapoint for year 2023, and updates the step code to use the latest available.

### Tasks
- [x] Snapshot, Meadow, Garden, Grapher
- [x] Update charts
- [x] Approve charts
- [x] Archive old dataset

### CODE REVIEW

File: dag/environment.yml [hunk 1]
```diff
@@ -1,12 +1,4 @@
 steps:
-  # NASA Ozone Watch
-  data://meadow/nasa/2023-03-06/ozone_hole_area:
-    - snapshot://nasa/2023-03-06/ozone_hole_area_p1.txt
-    - snapshot://nasa/2023-03-06/ozone_hole_area_p2.txt
-  data://garden/nasa/2023-03-06/ozone_hole_area:
-    - data://meadow/nasa/2023-03-06/ozone_hole_area
-  data://grapher/nasa/2023-03-06/ozone_hole_area:
-    - data://garden/nasa/2023-03-06/ozone_hole_area
   # UNEP Cumption of controlled substances
```
Comment (paarriagadap) [inline dag/environment.yml:2]: Just because I saw it: *Consumption

File: dag/archive/environment.yml
Comment (paarriagadap) [inline dag/archive/environment.yml:1]: By the way, there is a `conda environment file` text at the top of this file
image

**Review by paarriagadap — CHANGES_REQUESTED** [review]
Good! I request changes mostly because of the snapshots

File: snapshots/nasa/2024-05-31/ozone_hole_area_p1.txt.dvc [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    # Data product / Snapshot
+    title: Ozone hole area and concentration
+    description: |-
+      Annual maximum and Antarctic stratospheric ozone hole area, resultant from the emission of ozone-depleting substances.
+
+      Minimum and mean Southern Hemisphere daily ozone concentrations, measured in Dobson Units (DU).
+    date_published: "2024-05-16"
+
+    # Citation
+    producer: NASA Ozone Watch
```
[the remark this answers is not in the document]
Comment (lucasrodes) [reply snapshots/nasa/2024-05-31/ozone_hole_area_p1.txt.dvc:13]: they suggest we cite them like that

File: etl/steps/data/garden/nasa/2024-05-31/ozone_hole_area.meta.yml [hunk 3]
```diff
@@ -0,0 +1,59 @@
+dataset:
+  description: |
```
[the remark this answers is not in the document]
Comment (lucasrodes) [reply etl/steps/data/garden/nasa/2024-05-31/ozone_hole_area.meta.yml:2]: not really ー removed it

File: dag/archive/environment.yml
Comment (lucasrodes) [reply dag/archive/environment.yml:1]: not sure what this means? can't see it on github

File: etl/steps/data/garden/nasa/2024-05-31/ozone_hole_area.meta.yml [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+    variables:
+
+      max_hole_area:
+        title: Maximum ozone hole area
+        short_unit: "km²"
+        unit: "square kilometres"
+        description_short: |
+          The yearly maximum daily ozone hole area.
+        description_key:
+          - The ozone hole area is determined from total ozone satellite measurements. It is defined to be that region of ozone values below 220 Dobson Units (DU) located south of 40°S. Values below 220 DU represent anthropogenic ozone losses over Antarctica.
```
[the remark this answers is not in the document]
Comment (lucasrodes) [reply etl/steps/data/garden/nasa/2024-05-31/ozone_hole_area.meta.yml:20]: [there is one already](https://docs.google.com/document/d/1pwTrbEKOy4D3LPbpbNaivQyYVv2KTePpw-P4F5fMNjA/edit) ー i think it is used in the FASTT directly on admin

File: dag/archive/environment.yml
Comment (paarriagadap) [reply dag/archive/environment.yml:1]: It's weird, I click in VSCode and see this

https://github.com/owid/etl/assets/63430031/41e9cb35-0106-416c-8df4-ae5c57a4651c

Comment (lucasrodes) [reply dag/archive/environment.yml:1]: not sure what this means? can't see it on github.

i think vscode shows that warning bc the name of the file (`environment.yml`) might be reserved for some conda settings.

i don't use conda myself.

Comment (paarriagadap) [reply dag/archive/environment.yml:1]: And I don't see that for the other YAML files

File: etl/steps/data/garden/nasa/2024-05-31/ozone_hole_area.meta.yml (hunk 4 above)
[the remark this answers is not in the document]
Comment (paarriagadap) [reply etl/steps/data/garden/nasa/2024-05-31/ozone_hole_area.meta.yml:22]: Yes, it's just that they could also be useful there.

**Comment by lucasrodes** [comment]
thanks for your careful review, Pablo! <3

File: etl/steps/data/garden/nasa/2024-05-31/ozone_hole_area.meta.yml [hunk 5]
```diff
@@ -0,0 +1,61 @@
+definitions:
+  common:
+    presentation:
+      topic_tags:
+        - Ozone Layer
+
```
Comment (paarriagadap) [inline etl/steps/data/garden/nasa/2024-05-31/ozone_hole_area.meta.yml:6]: You also need to add processing_level

File: dag/archive/environment.yml
Comment (paarriagadap) [reply dag/archive/environment.yml:1]: Me neither in this machine. Weird.

**Review by paarriagadap — APPROVED** [review]

**Pull request merged**